docs/faq.rst

```diff
@@ -0,0 +1,102 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    In our :ref:`user guide <user_guide>` we provide links to the papers describing
+    Fairlearn's algorithms.
+    For a higher level view of the subject, please see the :ref:`further reading <further_reading>`
+    section.
+
+Why not just ignore the sensitive features?
+    Because ignoring the sensitive feature usually doesn't work.
+
+    To elaborate, information is often redundantly encoded in datasets, and machine learning
+    algorithms *will* ferret out these links (it is what they are designed to do).
```

Comment:
nit: can we use something other than "ferret"... figure?

Reply:
:-(